### PR TITLE
Enable JSF22StatelessViewTests for Faces 4.0 (EE10)

### PR DESCRIPTION
**Pull request by pnicolucci**
for #22369

### CODE REVIEW

File: dev/com.ibm.ws.jsf.2.2_fat/test-applications/JSF22StatelessView.war/src/com/ibm/ws/jsf22/fat/statelessview/beans/faces40/StatelessViewBean.java [hunk 1]
```diff
@@ -11,19 +11,21 @@
...
 
-import javax.faces.bean.ManagedBean;
+import javax.enterprise.context.RequestScoped;
 import javax.faces.component.UIViewRoot;
 import javax.faces.context.FacesContext;
+import javax.inject.Named;
 
-@ManagedBean
+@Named
+@RequestScoped
```
Comment (pnicolucci) [inline dev/com.ibm.ws.jsf.2.2_fat/test-applications/JSF22StatelessView.war/src/com/ibm/ws/jsf22/fat/statelessview/beans/faces40/StatelessViewBean.java:33]: Default ManagedBean scope is `@RequestScoped` default CDI scope is `@Dependent` so just adding a scope here explicitly to ensure the test is the same for EE10.

**Comment by pnicolucci** [comment]
#libby #build

**Comment by pnicolucci** [comment]
Java 17 build: https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_uAS00Fq1Ee2P_cwgExA7aA

**Comment by pnicolucci** [comment]
Java 17 tests were successful

**Review by isaacrivriv — APPROVED** [review]
Looks good, thanks for working on this!

**Comment by pnicolucci** [comment]
#libby #build

**Pull request merged**